### ISSUE TITLE
`plan` command panics if `init` wasn't run first

### ISSUE THREAD

**Issue by mkyc** [epiphany-platform/m-azure-basic-infrastructure#49]
`plan` command panics if `init` wasn't run first. This should be handled without panic but with clear message to user. 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x7b6cd6]

goroutine 1 [running]:
github.com/epiphany-platform/m-azure-basic-infrastructure/cmd.templateTfVars(0x0, 0x8ed2e0, 0xc00003da00)
	/go/src/github.com/epiphany-platform/m-azure-basic-infrastructure/cmd/common.go:63 +0x116
github.com/epiphany-platform/m-azure-basic-infrastructure/cmd.glob..func11(0xb15f20, 0xb4f830, 0x0, 0x0)
	/go/src/github.com/epiphany-platform/m-azure-basic-infrastructure/cmd/plan.go:50 +0x1d8
github.com/spf13/cobra.(*Command).execute(0xb15f20, 0xb4f830, 0x0, 0x0, 0xb15f20, 0xb4f830)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xb161c0, 0x406265, 0xc00005a058, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/epiphany-platform/m-azure-basic-infrastructure/cmd.Execute()
	/go/src/github.com/epiphany-platform/m-azure-basic-infrastructure/cmd/root.go:64 +0x2d
main.main()
	/go/src/github.com/epiphany-platform/m-azure-basic-infrastructure/main.go:8 +0x25
```

**Comment by mkyc** [comment]
Addressed in #66. 

> azbi docker run --rm -v $(pwd)/shared:/shared -e SUBSCRIPTION_ID=XXX -e CLIENT_ID=YYY -e CLIENT_SECRET=ZZZ -e TENANT_ID=KKK epiphanyplatform/azbi:dev plan   
> 2021-01-29T09:20:52Z FTL go/src/github.com/epiphany-platform/m-azure-basic-infrastructure/cmd/plan.go:47 > checkAndLoad failed error="state file does not exist, please run init first"

**Comment by przemyslavic** [comment]
Fixed.
The error message `state file does not exist, please run init first` is now more understandable.
```
docker run --rm -v /c/tmp/shared:/shared -e SUBSCRIPTION_ID=xxx -e CLIENT_ID=xxx -e CLIENT_SECRET=xxx -e TENANT_ID=xxx epiphanyplatform/azbi:develop plan
2021-02-18T12:21:42Z FTL go/src/github.com/epiphany-platform/m-azure-basic-infrastructure/cmd/plan.go:47 > checkAndLoad failed error="state file does not exist, please run init first"
```
